efficientdet/hparams_config.py

```diff
@@ -182,6 +182,7 @@ def default_detection_configs():
   h.jitter_max = 2.0
   h.autoaugment_policy = None
   h.use_augmix = False
+  h.grid_mask = False
```

Comment:
@mingxingtan Do i need to add configuration for gridmask like augmix?

Reply:
Thank you, @kartik4949 
Having this flag is good enough.  Do you find this augmentation helpful?

Reply:
@mingxingtan you can refer to the above links 
And please test this pr with your data or sample data ,Im short on resources so cant train but i have checked the augmentation for 2-3 images!

Reply:
ok, I will merge this PR and give it a try on COCO later.

Reply:
@mingxingtan yes please do try it with different hyperparameters :)